modal.js

```diff
@@ -2,14 +2,18 @@
   var modal = document.querySelector("#modal");
   var modalOverlay = document.querySelector("#modal-overlay");
   var closeButton = document.querySelector("#close-button");
-  var createCloud = document.querySelector("#create-goal");
+  var createGoal = document.getElementById("create-goal");
+
+  createGoal.addEventListener('mouseover', function() {
+    createGoal.classList = 'goal';
```

Comment:
This is kinda fragile, as it will overwrite any class that someone else tries to add.. maybe `createGoal.classList += ' goal'` or depending on your target platform
http://stackoverflow.com/questions/507138/how-do-i-add-a-class-to-a-given-element
```js
var element = document.getElementById("div1");
element.classList.add("otherclass");
```